src/gateways/GatewayKeyValuePairValueField.js

```diff
@@ -11,30 +11,28 @@ import React from 'react'
 import { dataTest } from '../dataTest'
 import i18n from '../locales'
 
-const { Field } = ReactFinalForm
+const { Field, useField } = ReactFinalForm
 const isStringWithLengthAtLeastOne = composeValidators(string, hasValue)
 
 export const createFieldGatewayKeyValuePairValueName = index =>
     `parameters[${index}].value`
 
-export const FIELD_GATEWAY_KEY_VALUE_PAIR_VALUE_LABEL = i18n.t('Value')
+export const GatewayKeyValuePairValueField = ({ index }) => {
+    const { input } = useField(`parameters[${index}]`)
```

Comment:
Since you want the value here, I think the useFormState hook is more appropriate (https://final-form.org/docs/react-final-form/api/useFormState).